### PR TITLE
Smoke tests - Fix the product selector in the FO

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix the product selector in the FO home page
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | URL_FO=YOUR_SHOP_URL TEST_PATH="smoke/crawlingFO" npm run specific-test 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/15206)

### CODE REVIEW

File: tests/puppeteer/pages/FO/home.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ module.exports = class Home extends CommonPage {
 
     // Selectors for home page
     this.logoHomePage = '#_desktop_logo';
-    this.productImg = '#content article:nth-child(%NUMBER) img';
+    this.productImg = '#content div:nth-child(%NUMBER) article img';
```
Comment (boubkerbribri) [inline tests/puppeteer/pages/FO/home.js]:
```suggestion
    this.productImg = '.products div:nth-child(%NUMBER) article img';
```
Maybe starting from list of products is better than content which contain carousel and some other things ?

Comment (SimonGrn) [reply tests/puppeteer/pages/FO/home.js]: I think it's better too

Comment (nesrineabdmouleh) [reply tests/puppeteer/pages/FO/home.js]: Yes, it's true, but I think that it's better to use an id than a class name in a selector
I propose to add an id for the products bloc, what do you think?

Comment (boubkerbribri) [reply tests/puppeteer/pages/FO/home.js]:
```suggestion
    this.productImg = '#content .products div:nth-child(%NUMBER) article img';
```

**Review by boubkerbribri — APPROVED** [review]
Looks good
Thanks @nesrineabdmouleh

**Review by matthieu-rolland — APPROVED** [review]
lgtm

**Pull request merged**